### ISSUE TITLE
Cuda 11 integration and roadmap

### ISSUE THREAD

**Issue by enricovillagrossi** [fzi-forschungszentrum-informatik/gpu-voxels#122]
Dear all,
Is it foreseen the support for CUDA 11? Is there a roadmap for the future developments?
Thanks,

Enrico

**Comment by cjue** [comment]
Yes, CUDA 11 compatibility is on my mind, I need to update the testing infrastructure with newer Ubuntu versions.

What Ubuntu version are you using, or are you on another platform?

**Comment by enricovillagrossi** [comment]
Thanks for the reply.
Currently I'm using Ubuntu 20.04.
